### PR TITLE
Update to reflect todays setup on the Windows 11 Developer PCs.

### PR DESCRIPTION
**Pull request by statnett-bao**
I've made some changes to make it easier to install the tools on the Windows 11 developer laptop I got from Statnett, which is:

1. Ignored the "galaxy"-directory that Ansible creates for storing its dependencies.
2. Updated the base-image of Dockerfile to *jammy* to reflect the current version of Ubuntu that is provided by WSL.
3. Merged the Ansible-commands into *bootstrap.sh* to lessen the amount of commands to execute (and replaced *apt* with *apt-get* to squash warning-messages about "unstability" [sic]).
4. Updated README.md to be more informative and reflect the base developer PC setup I got from Statnett. Also, I've added some commands to simplify installing WSL.

The changes has been tested on the Windows 11 developer laptop I was given from Statnett and by running **docker build .** multiple times.

### CODE REVIEW

**Comment by michaeloa** [comment]
Thank you. :)

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:focal
```
Comment (serpro69) [inline Dockerfile:1]: we don't fully support ubuntu 22.04 right now, it still has unresolved issues with e.g. phabricator and newer versions of php.

File: bootstrap.sh [hunk 2]
```diff
@@ -1,24 +1,25 @@
 #!/usr/bin/env bash
 # Bootstrap script for devxp-linux. Run this the first time you set up your WSL system.
 
-# Upgrade
-sudo apt update
+# Upgrade the distro.
+sudo apt-get update
```
Comment (serpro69) [inline bootstrap.sh:5]: isn't `apt` supposed to be a "substitute" for `apt-get`? Any reason we want to use `apt-get` explicitly ?

File: Dockerfile [hunk 3]
```diff
@@ -1,4 +1,4 @@
...
+FROM ubuntu:jammy
```
Comment (serpro69) [inline Dockerfile:1]: we don't fully support ubuntu 22.04 right now, it still has unresolved issues with e.g. phabricator and newer versions of php.

**Pull request merged**